content/guides/authentication/jwt/index.md

```diff
@@ -14,32 +14,35 @@ alias_paths:
 
 # JWT Auth
 
-Server-side authentication using JWT one of the most powerful ways to
+Server-side authentication using JSON Web Tokens (JWT) is the most common way to
 authenticate to the Box API. JWT is an [open standard](https://jwt.io/)
 designed to allow powerful server-to-server authentication.
 
 <ImageFrame border>
   ![The JWT flow](./jwt-flow.png)
 </ImageFrame>
 
-Server-side authentication using JSON Web Tokens (JWT) is only available to
-Custom Apps and Enterprise Integrations. It does not involve a user into the
-authorization flow and as such can be used to act on behalf of any user in an
-enterprise. JWT uses a public/private key pair verify the application's
-permissions.
+Server-side authentication using JWT is only available to the Custom Application
+[app type][app-type]. This authentication method does not require end-user
+interaction and, if granted the proper privileges, can be used
+to act on behalf of any user in an enterprise. 
 
-## JWT Restrictions
+There are two ways you can verify an application's permissions:
 
-Server-side authentication using JWT works by creating a claim on the
-application's server and then signing this using the application's secret key.
-In most cases, the claim is that the server is allowed to act on
-behalf of the Box application.
+- using a public and private key pair
+- using a client id and client secret (Client Credentials Grant)
```

Comment:
Would it not be nice again here to just make a new copy of this page for the client-credentials grant authentication? I don't see client credentials grant a sa subset of JWT.